python/ray/rllib/models/action_dist.py

```diff
@@ -69,6 +72,25 @@ def sampled_action_prob(self):
         """Returns the log probability of the sampled action."""
         return tf.exp(self.logp(self.sample_op))
 
+    @DeveloperAPI
+    @staticmethod
```

Comment:
```suggestion
    @classmethod
```

Reply:
I don't really understand this suggestion. Why do you think this should be a class method?

Reply:
Hm I guess staticmethod is fine, since you don't really need the class.